greedy_algorithms/kruskals_minimum_spanning_tree.cpp

```diff
@@ -1,69 +1,179 @@
 /**
  * @file
- * @brief [Kruskals Minimum Spanning
- * Tree](https://www.simplilearn.com/tutorials/data-structure-tutorial/kruskal-algorithm)
+ * @brief [Kruskal's Minimum Spanning Tree](https://www.simplilearn.com/tutorials/data-structure-tutorial/kruskal-algorithm)
  * implementation
  *
  * @details
- * _Quoted from
- * [Simplilearn](https://www.simplilearn.com/tutorials/data-structure-tutorial/kruskal-algorithm)._
+ * Kruskal’s algorithm is used to find the Minimum Spanning Tree (MST) for a connected weighted graph. 
+ * This algorithm sorts all edges by their weight and adds the smallest edge to the MST while avoiding cycles.
  *
- * Kruskal’s algorithm is the concept that is introduced in the graph theory of
- * discrete mathematics. It is used to discover the shortest path between two
- * points in a connected weighted graph. This algorithm converts a given graph
- * into the forest, considering each node as a separate tree. These trees can
- * only link to each other if the edge connecting them has a low value and
- * doesn’t generate a cycle in MST structure.
- *
- * @author [coleman2246](https://github.com/coleman2246)
+ * @author [MeetMendapara09](https://github.com/Meetmendapara09)
  */
 
-#include <array>     /// for array
-#include <iostream>  /// for IO operations
+#include <iostream>   // for IO operations
+#include <vector>     // for std::vector
+#include <algorithm>  // for std::sort
+#include <array>      // for std::array
+
+using namespace std;
```

Comment:
```suggestion
```
Doing `using namespace std` is bad practice. We've listed this on the contribution guidelines